build/scripts/deploy-msbuild.ps1

```diff
@@ -0,0 +1,50 @@
+# This script will ensure that the MSBuild toolset of the specified 
+# version is downloaded and deployed to Binaries\Toolset\MSBuild
+#
+# This is a no-op if it is already present at the appropriate version
+
+[CmdletBinding(PositionalBinding=$false)]
+param ($version = "0.1.7-alpha")
+
+Set-StrictMode -version 2.0
+$ErrorActionPreference="Stop"
+
+try {
+    
+    . (Join-Path $PSScriptRoot "build-utils.ps1")
+    $toolsDir = Join-Path $binariesDir "Toolset"
+    $scratchDir = Join-Path $toolsDir "Scratch"
+
+    Create-Directory $toolsDir
+    Create-Directory $scratchDir
+    pushd $toolsDir
+
+    $scratchFile = Join-Path $scratchDir "msbuild.txt"
+    $scratchVersion = Get-Content -raw $scratchFile -ErrorAction SilentlyContinue
+    if ($scratchVersion -eq ($version + [Environment]::NewLine)) { 
+        exit 0
+    }
+
+    $tempFileName = "msbuild-$version.zip"
+    $tempFile = Join-Path $scratchDir $tempFileName
+    if (-not (Test-Path $tempFile)) { 
+        $url = "https://jdashstorage.blob.core.windows.net/msbuild/msbuild-$version.zip" 
```

Comment:
It's my private azure subscription.  Was using it as a test bed for this change.  Now that I'm close to approval I broke this up into two NuGet packages so it will come through normal channels.